src/chigame/api/tests.py

```diff
@@ -0,0 +1,136 @@
+        """
+
+        for key in expected:
+            self.assertEqual(getattr(object, key), expected[key])
+
+    def test_create_game(self):
+        """
+        Ensure we can create a new game object.
+        """
+
+        # Create a game object
+        url = reverse("api-game-list")
+        data = {"name": "The Witcher 3", "description": "Geralt of Rivia", "min_players": 1, "max_players": 4}
+        response = self.client.post(url, data, format="json")
+        # Check that the game object was created correctly
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        game = Game.objects.get()
+        self.assertEqual(Game.objects.count(), 1)
+        self.check_equal(game, data)
+
+    def test_get_game(self):
+        """
+        Ensure we can get a game object.
+        """
+
+        # Create a game object
+        url = reverse("api-game-list")
+        data = {"name": "The Witcher 3", "description": "Geralt of Rivia", "min_players": 1, "max_players": 4}
+        response = self.client.post(url, data, format="json")
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # Get the game object
+        url = reverse("api-game-detail", kwargs={"pk": 1})
+        response = self.client.get(url, format="json")
+
+        # Check that the game object was retrieved correctly
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Game.objects.count(), 1)
+
+        game = Game.objects.get()
+        self.check_equal(game, data)
+
+    def test_update_game(self):
+        """
+        Ensure we can update a game object.
+        """
+
+        # Create a game object
+        url = reverse("api-game-list")
+        data = {"name": "The Witcher 3", "description": "Geralt of Rivia", "min_players": 1, "max_players": 4}
+        response = self.client.post(url, data, format="json")
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # Update the game object
+        url = reverse("api-game-detail", kwargs={"pk": 1})
+        data = {"name": "The Witcher 3: Wild Hunt", "max_players": 5}
+        response = self.client.patch(url, data, format="json")
+
+        # Check that the game object was updated correctly
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Game.objects.count(), 1)
+
+        game = Game.objects.get()
+        self.check_equal(
+            game,
+            {
+                "name": "The Witcher 3: Wild Hunt",
+                "description": "Geralt of Rivia",
+                "min_players": 1,
+                "max_players": 5,
+            },
+        )
+
+    def test_delete_game(self):
+        """
+        Ensure we can delete a game object.
+        """
+
+        # Create a game object
+        url = reverse("api-game-list")
+        data = {"name": "The Witcher 3", "description": "Geralt of Rivia", "min_players": 1, "max_players": 4}
```

Comment:
Given that you're reusing this same dictionary multiple times, you may want to create a helper factory like in the Django testing notes: 

https://canvas.uchicago.edu/courses/51666/files/10210480?wrap=1&fd_cookie_set=1

This will be especially relevant as the scale of the game related endpoints grows and you'll want more robust and easier to write tests.